backend/src/nodes/utils/ncnn_model.py

```diff
@@ -0,0 +1,524 @@
+from copy import deepcopy
+from io import BufferedReader
+import os
+from typing import Dict, List, Tuple, Union
+
+from json import load as jload
+import numpy as np
+from onnx import TensorProto
+import onnx.numpy_helper as onph
+from sanic.log import logger
+
+param_schema_file = os.path.join(
+    os.path.dirname(os.path.realpath(__file__)), "ncnn_param_schema.json"
+)
+with open(param_schema_file) as f:
+    param_schema = jload(f)
+
+DTYPE_FP32 = b"\x00\x00\x00\x00"
+DTYPE_FP16 = b"\x47\x6b\x30\x01"
```

Comment:
Do these have to be binary strings? Or could you put them in here as hex? (i.e. 0x00000000)

Reply:
Is there a reason to have them as numbers? Then they have to be converted every time they're written to file or buffer.

Reply:
I was gonna say for clarity, but it doesn't matter tbh